src/cocaine/burlak/metrics/procfs.py

```diff
@@ -275,3 +292,176 @@ def calc_load(total, used, cached):
             used = used - cached
 
         return clamp(used / float(total), .0, 1.0)
+
+
+class IfSpeed(ProcfsMetric):
+    """Read NIC speed from sysfs.
+
+    TODO(IfSpeed): Deprecated: it seems that there is no any portable way
+    to get real network speed bandwidth, e.g. for kvm
+        sysfs:/sys/class/network/*/speed
+    could be unavailable (is is useless here), or it could be set incorrectly
+    for virtual interfaces in container (arguable, needs investigation).
+    """
+    SYSFS_FNAME = 'speed'
+
+    def __init__(self, path):
+        """Init procfs network metric with specified path."""
+        super(IfSpeed, self).__init__(path)
+
+    @gen.coroutine
+    def read(self):
+        """Read netlink speed in Mbs."""
+        return IfSpeed._parse(self.read_nfirst_lines())
+
+    @staticmethod
+    def _parse(lines):
+        return int(lines[0])
+
+    @staticmethod
+    def make_path(prefix, iface):
+        """Construct sysfs path for iface speed file."""
+        return '{}/{}/{}'.format(prefix, iface, IfSpeed.SYSFS_FNAME)
+
+
+class Network(ProcfsMetric):
+    """Read network stat from /proc/net/dev.
+
+    TODO: it should be configurable interfaces name (prefix) or way to obtain
```

Comment:
Let's make it configurable right now. Both main interface name and its speed should be specified in config.

Reply:
Done, config properties:
 - netlink_speed_mbits
 - netlink_default_name